mmv1/third_party/terraform/services/apigee/resource_apigee_instance_update_test.go.erb

```diff
@@ -0,0 +1,312 @@
+<% autogen_exception -%>
+package apigee_test
+<% unless version == 'ga' -%>
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccApigeeInstance_updateConsumerAcceptList(t *testing.T) {
+	acctest.SkipIfVcr(t)
```

Comment:
Did this test pass locally? It's not recommended to skip vcr since that's what's running on git.

Reply:
Test was run with the GA provider instead of beta by mistake during local testing, so I fixed the test case.
Here is the output of a locally run test case.

```sh
❯ make testacc TEST=./google/services/apigee TESTARGS='-run=TestAccApigeeInstance_updateConsumerAcceptList'
TF_ACC=1 TF_SCHEMA_PANIC_ON_ERROR=1 go test ./google/services/apigee -v -run=TestAccApigeeInstance_updateConsumerAcceptList -timeout 240m -ldflags="-X=github.com/hashicorp/terraform-provider-google/version.ProviderVersion=acc"
=== RUN   TestAccApigeeInstance_updateConsumerAcceptList
=== PAUSE TestAccApigeeInstance_updateConsumerAcceptList
=== CONT  TestAccApigeeInstance_updateConsumerAcceptList
--- PASS: TestAccApigeeInstance_updateConsumerAcceptList (2595.84s)
PASS
ok  	github.com/hashicorp/terraform-provider-google/google/services/apigee	2597.180s
```

Please review them again.

Reply:
Could you remove this line and let VCR run this test?

Reply:
I fixed it.